src/transport_performance/gtfs/gtfs_utils.py

```diff
@@ -48,17 +58,20 @@ def bbox_filter_gtfs(
     _is_expected_filetype(
         pth=out_pth, param_nm="out_pth", check_existing=False
     )
-    _check_list(ls=bbox_list, param_nm="bbox_list", exp_type=float)
+    _type_defence(bbox, "bbox", (list, GeoDataFrame))
     for param in [units, crs]:
         if not isinstance(param, str):
             raise TypeError(f"Expected string. Found {type(param)} : {param}")
```

Comment:
Another additional comment on this.
It's possible that the tests could be removed (lines 29-34 in `test_gtfs_utils.py`) as we would be duplicating tests for `_type_defence()` at that point.  
While it isn't necessary to remove this test, it would reduce the overall number of tests (increasing efficiency of running the test suite)

Reply:
> Another additional comment on this. It's possible that the tests could be removed (lines 29-34 in `test_gtfs_utils.py`) as we would be duplicating tests for `_type_defence()` at that point. While it isn't necessary to remove this test, it would reduce the overall number of tests (increasing efficiency of running the test suite)

Will check on this one. Sounds like some good efficiencies spotted - thanks!